package.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "create-web3js-dapp",
+  "version": "0.0.0",
```

```suggestion
  "version": "0.1.0",
```
for initial dev version